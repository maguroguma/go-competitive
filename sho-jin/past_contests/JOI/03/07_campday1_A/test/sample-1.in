6
100
90
80
90
100
65
