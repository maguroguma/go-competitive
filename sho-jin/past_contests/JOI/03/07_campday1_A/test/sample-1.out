1
3
5
3
1
6
